gcampus/core/templates/gcampuscore/components/saprobic_index.html

```diff
@@ -0,0 +1,75 @@
+{% load i18n measurement %}
+<div class="bg-light me-2">
+    <div class="my-auto p-4">
+
+        <div class="row">
+            <div class="col-9">
+                <div class="text-secondary fw-bold pb-2">
+                    {% translate "Saprobic Index" %}
+                </div>
+                {% if measurement.saprobic_index.validity >= 0.25 %}
+                    <div class="index-classification display-1 fw-bolder saprobic-class-{{ measurement.saprobic_index.classification }}">
+                        {% translate "Class" %} {{ measurement.saprobic_index.classification }}
+                    </div>
+                    <div class="index-classification display-4 fw-bolder bach-class-{{ measurement.saprobic_index.classification }}">
+                        {{ measurement.saprobic_index.description }}
+                    </div>
+                {% else %}
+                    <div class="index-classification display-1 fw-bolder class-unknown ">
+                        {% translate "N/A" context "no value provided (water name, comment, etc)" as not_any %}
+                    </div>
+                    <div class="invisible display-4">
+                        {% translate "N/A" context "no value provided (water name, comment, etc)" as not_any %}
+                    </div>
+                {% endif %}
+            </div>
+            <div class="col-3">
+                <div class="display-1 text-end text-gray pe-2">
+                    {% include "gcampuscore/icons/bug-fill.html" with extra_class="flex-shrink-0" %}
+                </div>
+            </div>
+        </div>
+        <div class="pt-4 row">
+            <div class="col-4">
+                <div class="text-secondary fw-light">
+                    {% translate "Value" %}
+                </div>
+                <div class="fw-bold">
+                    {% if measurement.saprobic_index.validity >= 0.25 %}
+                        {{ measurement.saprobic_index.value|floatformat:"2" }}
+                    {% else %}
+                        {% translate "N/A" context "no value provided (water name, comment, etc)" as not_any %}
```

Comment:
Same problem as in `gcampus/core/templates/gcampuscore/components/bach_index.html`.